### ISSUE TITLE
Sveltia cms UI not loading, no errors

### ISSUE THREAD

**Issue by moookino** [privatemaker/headless-cms#2]
Hi @privatemaker 
I followed steps in README and added config and params yamls to the config/_defaults dir, I didn't put anything into hugo.yaml in the home/src folder to keep all the configs in one place
So there's couple findings:
config.yaml - it wasn't enough to have only basic things you listed, but also params part with headless_cms and backed info, otherwise admin page was saying I didn't choose any supported CMS, so I ended up just having params config in both files, see below:
```
module:
  imports:
  - path: headless-cms

outputs:
  home:
  - HTML
  - RSS
  - HeadlessCMSConfig
media_folder: static/images
public_folder: /images
params:
  headless_cms:
    engine: "sveltia"
    site_url: "https://redacted.org"
    backend:
      name: gitlab
      repo: "redacted"
      auth_type: pkce # Required for pkce
      app_id: # "redacted"Application ID from your GitLab settings
    collections:
      - label: "Blog"
        name: "blog"
        folder: "content/posts"
        create: true
        fields:
          - {label: "Title", name: "title", widget: "string"}
          - {label: "Publish Date", name: "date", widget: "datetime"}
          - {label: "Featured Image", name: "thumbnail", widget: "image"}
          - {label: "Body", name: "body", widget: "markdown"}
```

So now sveltia is loaded fine in the network and I don't see any errors in console, but I don't see any UI loaded
![raw](https://github.com/user-attachments/assets/bee62cf1-8ddf-48f5-a68f-528e1ba7c427)
there's no error in console
![image](https://github.com/user-attachments/assets/5ff71faa-eda9-44af-bef0-8c42e7967980)

Can you help to understand what's wrong there?

Also I've tried decap, but it also fails to detect right path for config.yaml
![raw](https://github.com/user-attachments/assets/25e3d74d-328e-4de7-ba40-0e4239493ae8)

**Comment by richtong** [comment]
I have the same problem. Looking at the code it seems like it is just loading the sveltia.js file. I'll look at the sveltia issue

**Comment by privatemaker** [comment]
This issue was caused because Sveltia refactored how the JS file was compiled to no longer require `type="module"` in [v0.21.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.21.0). 

I reproduced the problem and fixed in in [headless-cms:0.1.1](https://github.com/privatemaker/headless-cms/releases/tag/v0.1.1).

Apologies for slow response here.

**Comment by richtong** [comment]
Thanks!

Sent via Superhuman iOS ( ***@***.*** )
